addons/hearing/XEH_postInit.sqf

```diff
@@ -9,31 +9,31 @@ GVAR(deafnessDV) = 0;
 GVAR(deafnessPrior) = 0;
 GVAR(volume) = 1;
 GVAR(playerVehAttenuation) = 1;
-
-GVAR(beep) = false;
-GVAR(beep2) = false;
-GVAR(time2) = 0;
 GVAR(time3) = 0;
-GVAR(time4) = 0;
+GVAR(damageCoefficent) = 1;
 
 ["SettingsInitialized", {
+    TRACE_1("settingInit",GVAR(EnableCombatDeafness));
+    // Only run PFEH and install event handlers if combat deafness is enabled
+    if (!GVAR(EnableCombatDeafness)) exitWith {};
+
```

Comment:
@PabstMirror, we should probably call `fnc_updateHearingProtection` here once, in case `SettingsInitialized` arrive when the player is already fully formed.